docs/jdbc-mariadb.md

```diff
@@ -1,163 +1,3 @@
-IAM principal must have the iam.serviceAccounts.getAccessToken permission or
-the role roles/iam.serviceAccounts.serviceAccountTokenCreator.
-
-Example:
-```java
-// Set up URL parameters
-String jdbcURL = String.format("jdbc:mariadb://ignoreme:123/%s", DB_NAME);
-Properties connProps = new Properties();
-connProps.setProperty("user", "mysql-iam-user@gmail.com");
-connProps.setProperty("password", "password");
-connProps.setProperty("sslmode", "disable");
-connProps.setProperty("socketFactory", "com.google.cloud.sql.mariadb.SocketFactory");
-connProps.setProperty("cloudSqlInstance", "project:region:instance");
-connProps.setProperty("enableIamAuth", "true");
-connProps.setProperty("cloudSqlTargetPrincipal", "mysql-iam-user@gmail.com");
-
-// Initialize connection pool
-HikariConfig config = new HikariConfig();
-config.setJdbcUrl(jdbcURL);
-config.setDataSourceProperties(connProps);
-config.setConnectionTimeout(10000); // 10s
-
-HikariDataSource connectionPool = new HikariDataSource(config);
-```
-
-In addition, the `cloudSqlDelegates` property controls impersonation delegation. 
-The value is a comma-separated list of service accounts containing chained 
-list of delegates required to grant the final access_token. If set,
-the sequence of identities must have "Service Account Token Creator" capability
-granted to the preceding identity. For example, if set to 
-`"serviceAccountB,serviceAccountC"`, the application default credentials must 
-have the Token Creator role on serviceAccountB. serviceAccountB must have
-the Token Creator on serviceAccountC. Finally, C must have Token Creator on 
-`cloudSqlTargetPrincipal`. If unset, the application default credential principal
-must "Service Account Token Creator" capability granted that role on the
-targetPrincipal service account.
-
-
-For example:
-```java
-connProps.setProperty("cloudSqlTargetPrincipal", "TARGET_SERVICE_ACCOUNT");
-connProps.setProperty("cloudSqlDelegates", "SERVICE_ACCOUNT_1,SERVICE_ACCOUNT_2");
-```
-
-In this example, the environment's application default principal impersonates
-SERVICE_ACCOUNT_1 which impersonates SERVICE_ACCOUNT_2 which then
-impersonates the TARGET_SERVICE_ACCOUNT.
-
-## Examples
-
-Examples for using the Cloud SQL JDBC Connector for MySQL/MariaDB can be found by looking at the integration tests in this repository.
-* [Usage example](../jdbc/mariadb/src/test/java/com/google/cloud/sql/mariadb/JdbcMariaDBIntegrationTests.java)
-
-## Reference Documentation
-* [Connecting to Cloud SQL from App Engine Standard](https://cloud.google.com/sql/docs/mysql/connect-app-engine-standard)
-* [Connecting to Cloud SQL from App Engine Flexible](https://cloud.google.com/sql/docs/mysql/connect-app-engine-flexible)
-* [Connecting to Cloud SQL from Cloud Functions](https://cloud.google.com/sql/docs/mysql/connect-functions)
-* [Connecting to Cloud SQL from Cloud Run](https://cloud.google.com/sql/docs/mysql/connect-run)
+See [jdbc.md](./jdbc.md)
```

Comment:
Let's just delete these files entirely and link to sections as needed from the README.